.yarn/versions/ddf3a56f.yml

```diff
@@ -0,0 +1,2 @@
+releases:
+  "@yarnpkg/sdks": minor
```

Comment:
I chose this as a minor release because `yarn dlx @yarnpkg/sdks vscode` theoretically has to be rerun for packages that use svelte-language-tools. This makes it feel like more than a bugfix, however it's still "backwards compatible" because this should only be needed to fix broken projects so it's not a major breaking change either.

Let me know if the versioning isn't supposed to work that way!